composer.json

```diff
@@ -58,7 +58,6 @@
         }
     },
     "replace": {
-        "typo3_console": "self.version",
         "typo3-ter/typo3-console": "self.version"
```

Comment:
oh wow. good catch. no idea what else need to be fixed and whether the extension release process would still work.

Reply:
Maybe add the extension key to the extra section, just to be sure?

Reply:
Yeah, thought about the extension-key too but wasn't that added with v2? Did not check but I guess 7.6 will work with v1 of the cms-composer-installers only. Anyway it should be covered by the fallback to the composer package name which will be properly converted as far as I could see.
Will check this quickly...

Reply:
7.6.49 ELTS: "typo3/cms-composer-installers": "~1.3.4" and this version has already the ext-key, so I'm going to add it now...
https://github.com/TYPO3/CmsComposerInstallers/blob/1.3.x/src/Installer/ExtensionInstaller.php#L264